rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/widget/CovidRDTBarcodeFactory.java

```diff
@@ -41,10 +42,9 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
 
                 String[] individualVals = splitCSV(barcodeVals);
                 populateRelevantFields(individualVals);
-                moveToNextStep(convertDate(individualVals[1], "YYYY-MM-dd"));
-            } catch (JSONException e) {
-                Timber.e(e);
-            } catch (ParseException e) {
+                final int sensorTriggerIndex = 4;
```

Comment:
Could you move this to a global constant and share replace this as well  https://github.com/onaio/rdt-standard/blob/a454133b83391cefd15a9f78e6080db225065a79/rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/widget/CovidRDTBarcodeFactory.java#L69?

Reply:
Also move the `individualVals[1]` index to a global constant and replace this as well https://github.com/onaio/rdt-standard/blob/a454133b83391cefd15a9f78e6080db225065a79/rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/widget/CovidRDTBarcodeFactory.java#L66

Reply:
fixed.